src/components/Articles/EditArticle.js

```diff
@@ -0,0 +1,160 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import Loader from 'react-loader';
+import {
+  fetchSpecificArticle,
+  updateArticle,
+} from '../../actions/articleActions';
+import Article from './Article';
+import CreateArticleForm from './CreateArticleForm';
+
+export class EditArticle extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      description: '',
+      body: '',
+      isEditing: false,
+    };
+  }
+
+  componentDidMount() {
+    const { fetchSpecificArticle } = this.props;
+    const {
+      match: {
+        params: { slug },
+      },
+    } = this.props;
+    fetchSpecificArticle(slug);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const { articlePayload } = nextProps;
+    if (nextProps.articlePayload !== this.props.articlePayload) {
```

Comment:
Did you intentionally ignore this linting error?

Reply:
This was skipped accidentally. It has been restructured.